clients/drcachesim/simulator/caching_device_stats.h

```diff
@@ -57,9 +57,19 @@
 namespace dynamorio {
 namespace drmemtrace {
 
+// Enumerates the different types of cache line invalidation events that
```

Comment:
Do we want to be using Doxygen? I see some Doxygen in some dr$sim headers, but it is not consistently used, and I guess it's not sent to html for the website as `policy_fifo_t` doesn't show up and it has Doxygen: https://dynamorio.org/search?query=policy_fifo_t.

Given that dr$sim is growing maybe it's worth moving toward publicly documenting its extension interfaces? Use Doxygen here and for the entries below even if it's not yet used for html?

Reply:
I changed the comments so that they're shown on our pages. Also added some other headers in the simulator directory  to the docs.